build.sbt

```diff
@@ -45,18 +47,18 @@ lazy val magnum = project
     Test / fork := true,
     publish / skip := false,
     libraryDependencies ++= Seq(
-      "org.scalameta" %% "munit" % "0.7.29" % Test,
+      "org.scalameta" %% "munit" % "1.0.2" % Test,
       "com.dimafeng" %% "testcontainers-scala-munit" % testcontainersVersion % Test,
       "com.dimafeng" %% "testcontainers-scala-postgresql" % testcontainersVersion % Test,
-      "org.postgresql" % "postgresql" % "42.6.0" % Test,
+      "org.postgresql" % "postgresql" % "42.7.4" % Test,
       "com.dimafeng" %% "testcontainers-scala-mysql" % testcontainersVersion % Test,
-      "mysql" % "mysql-connector-java" % "8.0.32" % Test,
-      "com.h2database" % "h2" % "2.1.214" % Test,
+      "mysql" % "mysql-connector-java" % "8.0.33" % Test,
```

Comment:
It looks like this dep was moved to `"mysql" % "mysql-connector-j"` and the latest version is 9.0.0, can you please update?

https://dev.mysql.com/doc/connectors/en/connector-j-installing-maven.html

Reply:
Done. Thanks for spotting this!